protobuf/event/event.proto

```
@@ -1,39 +1,43 @@
 syntax = "proto3";
 
-package protobuf.event;
+package event;
```

Buf (1.55.1)

Files with package "event" must be within a directory "event" relative to root but were in directory "protobuf/event".

(PACKAGE_DIRECTORY_MATCH)